app/webpack/observations/show/ducks/observation.js

```diff
@@ -796,7 +796,8 @@ export function addID( taxon, options = { } ) {
     let observationTaxon = o.taxon;
     if (
       o.preferences.prefers_community_taxon === false
-      || (o.user.preferences.prefers_community_taxa === false && !o.preferences.prefers_community_taxon === true)
+      || (o.user.preferences.prefers_community_taxa === false 
+      && o.preferences.prefers_community_taxon === null)
```

Comment:
Sorry for the ambiguity, I was talking about this second commit, where I think I addressed both your original comments.

Reply:
Ah, the fault is mine. Sorry for not seeing that.